query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -29,7 +29,7 @@ sealed trait LogicalPlan {
       case l: LabelValues              => l.copy(filters = filters)
       case n: LabelNames               => n.copy(filters = filters)
       case s: SeriesKeysByFilters      => s.copy(filters = filters)
-      case c: TopkCardinalities        => c.copy()
+      case c: TsCardinalities          => c.copy()
```

Comment:
No need to copy the object as its immutable and there are no changes to make.

Reply:
Updated